src/com/seafile/seadroid2/ui/adapter/SeafItemAdapter.java

```diff
@@ -72,10 +86,23 @@ public boolean isEmpty() {
      * <p>
      * This method should be called after the download folder button was clicked.
      * 
-     * @param downloadTaskInfos
+     * @param newList
      */
-    public void setDownloadTaskList(List<DownloadTaskInfo> downloadTaskInfos) {
-        this.mDownloadTaskInfos = downloadTaskInfos;
```

Comment:
see above for my argument. this is all to avoid the issue mentioned in https://github.com/haiwen/seadroid/pull/267#issuecomment-74365872 to avoid repeated poking of the server in case the thumbnail could not be fetched.